### ISSUE TITLE
nfc-lab v2.1.0 Console windows to be removed and moved to Application debug/info windows

### ISSUE THREAD

**Issue by bvernoux** [josevcm/nfc-laboratory#5]
Actual version nfc-lab v2.1.0 (or before) display a console windows (in addition to main application)
It will be great to remove that window and display those debugs in a special tab (after events to call it debug or info ...)
![image](https://user-images.githubusercontent.com/432709/134768989-f927330b-3828-4a11-8eca-5a52446fb149.png)

**Comment by josevcm** [comment]
Yes, this is already planned for the next release version, in fact it is only necessary to change a macro in the Logger.cpp file and rebuild.

**Comment by bvernoux** [comment]
We can close this issue as it is done in latest version https://github.com/josevcm/nfc-laboratory/releases/tag/2.4.0
